xbmc/platform/posix/filesystem/SMBDirectory.cpp

```diff
@@ -330,3 +344,67 @@ bool CSMBDirectory::Exists(const CURL& url2)
   return S_ISDIR(info.st_mode);
 }
 
+bool CSMBDirectory::GetServerList(CFileItemList &items)
```

Comment:
```suggestion
bool CSMBDirectory::GetServerList(CFileItemList& items)
```

please follow our coding guidelines
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types